scripts/cleanup_old_vms.py

```diff
@@ -29,9 +30,16 @@ def parse_cmd_line():
         '(varies by provider, default 24)')
     parser.add_argument('--provider', dest='providers', action='append', default=None,
         help='Provider(s) to inspect, can be used multiple times', metavar='PROVIDER')
-    parser.add_argument('text_to_match', nargs='*', default=['^test_', '^jenkins', '^i-'],
+    parser.add_argument('text_to_match', nargs='*', default=['^test', '^jenkins', '^i-'],
```

Comment:
@psav, Just to Note: this was changed long back on yamls/Jenkins jobs. It was decided to delete any vm's starting with "test", as it was observed that there were many temporary vm's on providers like "test_", "testamogh", "testdeletethis" etc.

I can revert it here as its default values, Let Jenkins calls do the purpose?

I can revert it in the script